src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -141,8 +141,8 @@ public final class InlineConfigParser {
      */
     private static final Set<String> SUPPRESSED_CHECKS = new HashSet<>(Arrays.asList(
             "com.puppycrawl.tools.checkstyle.checks.regexp.RegexpCheck",
-            "com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyForInitializerPadCheck",
-            "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck"));
+            "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck",
+            "com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck"));
```

Comment:
Why are we adding `SummaryJavadocCheck`?

Reply:
It appears there may have been a slight oversight on my part, as I was processing 2 checks concurrently. I have made the new changes and `SummaryJavadocCheck` has been removed in this PR - https://github.com/checkstyle/checkstyle/pull/13963

Reply:
done